src/server/config/schema.js

```diff
@@ -52,6 +52,7 @@ module.exports = () => Joi.object({
       disableProtection: Joi.boolean().default(false),
       token: Joi.string().optional().notes('Deprecated')
     }).default(),
+    staticDirectoryListings: Joi.boolean().default(Joi.ref('$dev')),
```

Comment:
What does this line do? Make it so that defaults to `true` when in development and `false` otherwise?